src/operation-node/create-trigger-node.ts

```diff
@@ -0,0 +1,77 @@
+import { freeze } from '../util/object-utils.js'
+import { OperationNode } from './operation-node.js'
+import { TableNode } from './table-node.js'
+import { IdentifierNode } from './identifier-node.js'
+import { QueryNode } from './query-node.js'
+import { TriggerEventNode } from './trigger-event-node.js'
+import { TriggerOrderNode } from './trigger-order-node.js'
+import { FunctionNode } from './function-node.js'
+
+export type TriggerTime = 'after' | 'before' | 'instead of'
+
+export type CreateTriggerNodeParams = Omit<
+  CreateTriggerNode,
+  'kind' | 'name' | 'queries'
+>
+
+export interface CreateTriggerNode extends OperationNode {
+  readonly kind: 'CreateTriggerNode'
+  readonly name: IdentifierNode
+  readonly queries?: ReadonlyArray<QueryNode>
```

Comment:
`QueryNode` means only `select`, `insert`, `update`, `delete` queries are allowed. Which is not true for e.g. [MySQL](https://dev.mysql.com/doc/refman/8.0/en/stored-program-restrictions.html#stored-routine-sql-restrictions).

Reply:
> Which is not true for e.g. [MySQL](https://dev.mysql.com/doc/refman/8.0/en/stored-program-restrictions.html#stored-routine-sql-restrictions)

We could offer raw queries in addition to the traditional queries.